backend_python/backend/town.py

```diff
@@ -1,15 +1,26 @@
-from backend_python.backend.player import Player
+try:
+	from backend_python.backend.player import Player
+except ModuleNotFoundError:
+	from player import Player
```

Comment:
not sure why this is needed but probably worth investigating how local imports work and if we really need to do it the flask way, i have a feeling this is not really needed